src/software/pipeline/main_LdrToHdrMerge.cpp

```diff
@@ -175,7 +176,26 @@ int aliceVision_main(int argc, char** argv)
         }
     }
     std::vector<std::shared_ptr<sfmData::View>> targetViews;
-    hdr::selectTargetViews(targetViews, groupedViews, offsetRefBracketIndex);
+
+    const int middleIndex = usedNbBrackets / 2;
+    const int targetIndex = middleIndex + offsetRefBracketIndex;
+    const bool isOffsetRefBracketIndexValid = (targetIndex >= 0) && (targetIndex < usedNbBrackets);
+
+    const fs::path targetIndexFilepath(fs::path(inputResponsePath).parent_path() / (std::string("exposureRefIndex.txt")));
+
+    if (!fs::is_regular_file(targetIndexFilepath) && !isOffsetRefBracketIndexValid)
```

Comment:
When you have images with no bracketing, the program exits here with an error since `isOffsetRefBracketIndexValid = false`. If `nbBrackets = 1`, it will always fail.

There is a `byPass` parameter that is set to `true` when there's no bracketing just above. Maybe it should be used here, just as it a bit further down?

Reply:
It will fail if the manual parameter is out of range AND if no file can be read. The calibration node is supposed to write a file even if no bracketing is used.

Reply:
I will retest it but on Friday afternoon, I was always entering the "if" with my images (with no bracketing).

Reply:
I will have a look on the calibration node to check its behavior.

Reply:
Without any bracketing the sampling node does not provide any samples and as a consequence the calibration node cannot generate the expected file. I think, your first suggestion is the good thing to do.